### PR TITLE
Update to 2.4, add PostgreSQL 10

### LINKED ISSUE #62: Postgres 10 releae?
Hi,

I'm wondering if you are planning to release a version of this image based on the newly released PostgreSQL 10.

Thanks!

### PR DESCRIPTION
**Pull request by tdamsma**
Due to the problem that the postgis package is not versioned with a release number in the name (see [here](https://github.com/appropriate/docker-postgis/issues/26)), the version constraint for the postgis package  (which contains the `pgsql2shp`,`shp2pgsql`, `raster2pgsql`, and `shp2pgsql-gui` userland binaries)o. The 9.x-2.3 images thus have postgis 2.3 installed, but these binaries are from postgis 2.4

Fixes #62

### CODE REVIEW

File: Dockerfile.template [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
       && apt-get install -y --no-install-recommends \
            postgresql-$PG_MAJOR-postgis-$POSTGIS_MAJOR=$POSTGIS_VERSION \
            postgresql-$PG_MAJOR-postgis-$POSTGIS_MAJOR-scripts=$POSTGIS_VERSION \
-           postgis=$POSTGIS_VERSION \
+           postgis\
```
Comment (md5) [inline Dockerfile.template]: Can you put the space back: `postgis \`

File: .travis.yml [hunk 2]
```diff
@@ -15,6 +16,7 @@ env:
   - VERSION=9.3-2.3 VARIANT=alpine
   - VERSION=9.2-2.3
   - VERSION=9.2-2.3 VARIANT=alpine
+  - VERSION=10-2.4
```
Comment (md5) [inline .travis.yml]: What about Alpine variants?

Comment (md5) [reply .travis.yml]: I'm taking a stab at updating this myself (assuming you didn't uncheck the checkbox that lets me push to your branch).

Comment (tdamsma) [reply .travis.yml]: Great, `Allow edits from maintainers` is still checked

**Pull request closed without merging**